buildspec-localmodetests.yml

```diff
@@ -11,16 +11,5 @@ phases:
 
       # local mode tests
       - start_time=`date +%s`
-      - HAS_MATCHING_CHANGES_OUTPUT=$(has-matching-changes "tests/" "src/*.py" "setup.py" "setup.cfg" "buildspec.yml")
-      - |
-        if [ "$HAS_MATCHING_CHANGES" = "Changes Found" ] ; then
-          tox -e py36 -- tests/integ -m local_mode --durations 50
-        fi
-      - ./ci-scripts/displaytime.sh 'py36 local mode' $start_time
-
-      - start_time=`date +%s`
-      - |
-        if [ "$HAS_MATCHING_CHANGES" = "Changes Found" ] ; then
-          tox -e py27 -- tests/integ -m local_mode --durations 50
-        fi
-      - ./ci-scripts/displaytime.sh 'py27 local mode' $start_time
+      - execute-command-if-has-matching-changes "tox -e py27,py36 -- tests/integ -m local_mode --durations 50" "tests/" "src/*.py" "setup.py" "setup.cfg" "buildspec.yml"
```

Comment:
(optional) I'd be tempted to save the command in a variable to make it look a little cleaner